docs/releases/6.0.txt

```diff
@@ -269,6 +269,12 @@ Miscellaneous
 Features deprecated in 6.0
 ==========================
 
+Email
+-----
```

Comment:
Wasn't sure whether to put this in the "Email" or "Settings" section.

Reply:
Maybe `Settings`? Given that this is a settings change but there's no actual behavior change.  But either seems fine.

Reply:
I couldn't find a "Settings" section in past release notes. It seems like settings changes are either covered under their associated topic ([example](https://docs.djangoproject.com/en/4.2/releases/4.2/#custom-file-storages)) or under "Miscellaneous." Since there are a bunch of other email deprecations queuing up for 6.0, I think "Email" is probably better than "Miscellaneous."